compendium/modules/w05-classes-exercise.tex

```diff
@@ -215,8 +215,6 @@
 \input{generated/quiz-w05-case-class-taskrows-generated.tex}
 \end{ConceptConnections}
 
-\TODO \verb|Warning: Always false| visas inte i Scala 3, till skillnad från Scala 2.x. Ta bort?
```

Comment:
Yes, warning has been changed to an error, but good to have some relevant exercise showing the error such as
```
scala> "x" == 1
1 |"x" == 1
  |^^^^^^^^
  |Values of types String and Int cannot be compared with == or !=

```

Reply:
I changed the exercise to be an assignment instead of equality check, as explained in the comments above. I can revert the change if the compiler actually gives an error instead of `false`.